js/components/controlBar.js

```diff
@@ -317,103 +319,146 @@ var ControlBar = React.createClass({
+            onClick={this.handleVolumeIconClick}
+            onMouseUp={this.blurOnMouseUp}
+            onMouseOver={this.highlight}
+            onMouseOut={this.removeHighlight}
+            tabIndex="0"
+            aria-label={volumeAriaLabel}>
+            <Icon {...this.props} icon={volumeIcon} ref="volumeIcon"
+              style={this.props.skinConfig.controlBar.iconStyle.inactive} />
+            <Tooltip enabled={isTooltipEnabled}
+              text={this.props.controller.state.volumeState.muted ? Utils.getLocalizedString(this.props.language, CONSTANTS.SKIN_TEXT.UNMUTE, this.props.localizableStrings) : Utils.getLocalizedString(this.props.language, CONSTANTS.SKIN_TEXT.MUTE, this.props.localizableStrings)}
+              responsivenessMultiplier={this.responsiveUIMultiple} bottom={this.responsiveUIMultiple * this.props.skinConfig.controlBar.height} alignment={alignment}>
+            </Tooltip>
+          </button>
+          {volumeControls}
+        </div>
+      }).bind(this),
+
+      "timeDuration": (function (alignment) {
+        return <a className="oo-time-duration oo-control-bar-duration" style={durationSetting} key="timeDuration">
+          <span>{playheadTimeContent}</span>{totalTimeContent}
+        </a>
+      }).bind(this),
+
+      "flexibleSpace": (function (alignment) { return <div className="oo-flexible-space oo-control-bar-flex-space" key="flexibleSpace"></div> }).bind(this),
+
+      "moreOptions": (function (alignment) {
+        return <a className="oo-more-options oo-control-bar-item"
+          onClick={this.handleMoreOptionsClick} key="moreOptions" aria-hidden="true">
+          <Icon {...this.props} icon="ellipsis" style={dynamicStyles.iconCharacter}
+            onMouseOver={this.highlight} onMouseOut={this.removeHighlight} />
+          <Tooltip enabled={isTooltipEnabled} responsivenessMultiplier={this.responsiveUIMultiple} bottom={this.responsiveUIMultiple * this.props.skinConfig.controlBar.height}
+            text={Utils.getLocalizedString(this.props.language, CONSTANTS.SKIN_TEXT.MORE_OPTIONS, this.props.localizableStrings)} alignment={alignment}>
+          </Tooltip>
+        </a>
+      }).bind(this),
+
+      "quality": (function (alignment) {
+        return <div className="oo-popover-button-container" key="quality">
           {videoQualityPopover}
           <a className={qualityClass} onClick={this.handleQualityClick} style={selectedStyle} aria-hidden="true">
             <Icon {...this.props} icon="quality" style={dynamicStyles.iconCharacter}
-              onMouseOver={this.highlight} onMouseOut={this.removeHighlight}/>
+              onMouseOver={this.highlight} onMouseOut={this.removeHighlight} />
+            <Tooltip enabled={isTooltipEnabled} text={Utils.getLocalizedString(this.props.language, CONSTANTS.SKIN_TEXT.VIDEO_QUALITY, this.props.localizableStrings)} bottom={this.responsiveUIMultiple * this.props.skinConfig.controlBar.height} alignment={alignment}
```

Comment:
Minor thing, but we should standardize an order for the tooltip properties. Each of these buttons have the tooltip properties in a different order.